os_stub/spdm_device_secret_lib_sample/lib.c

```diff
@@ -28,6 +28,771 @@
 #include "library/memlib.h"
 #include "spdm_device_secret_lib_internal.h"
 
+#ifndef LIBSPDM_PRIVATE_KEY_USE_PEM
+#define LIBSPDM_PRIVATE_KEY_USE_PEM 0
```

Comment:
I think we need maintain the default behavior to 1.

Reply:
Thanks. I have changed the default value.